pages/admin/badges-add.js

```diff
@@ -479,4 +481,4 @@ export class AdminBadgesAdd extends Component {
   }
 }
 
-export default connect(['authenticatedUser', 'error', 'badges', 'admin'], actions)(AdminBadgesAdd)
+export default connect(['authenticatedUser', 'error', 'badges', 'admin', 'notification'], actions)(AdminBadgesAdd)
```

Comment:
```suggestion
export default connect(['authenticatedUser', 'error', 'badges', 'admin'], actions)(AdminBadgesAdd)
```

The 'notification' property of state isn't needed.